web/apps/labelstudio/src/pages/ExportPage/ExportPage.jsx

```diff
@@ -1,15 +1,31 @@
-import { useEffect, useRef, useState } from "react";
+import { useEffect, useRef, useState, useCallback } from "react";
 import { useHistory } from "react-router";
 import { Button } from "@humansignal/ui";
+import {
+  IconWarningCircleFilled,
+  IconTerminal,
+  IconCode,
+  IconBook,
+  IconExternal,
+  IconCopyOutline,
+} from "@humansignal/icons";
 import { Form, Input } from "../../components/Form";
 import { Modal } from "../../components/Modal/Modal";
 import { Space } from "../../components/Space/Space";
 import { useAPI } from "../../providers/ApiProvider";
 import { useFixedLocation, useParams } from "../../providers/RoutesProvider";
 import { cn } from "../../utils/bem";
-import { isDefined } from "../../utils/helpers";
+import { isDefined, copyText } from "../../utils/helpers";
 import "./ExportPage.scss";
 
+// Community Edition exports run synchronously in a single HTTP request.
+// Large exports can exceed typical proxy timeouts, so we warn early and link to alternatives.
+const LARGE_EXPORT_TASK_THRESHOLD = 5000;
```

Comment:
what is 5000? is it seconds? why 5000?
if it's seconds you can add to comment plus split it into parts for readability like `60 * 60 * 1.5; // 1.5 hours`